src/locale/localed.c

```diff
@@ -32,7 +32,7 @@
 #include "strv.h"
 #include "user-util.h"
 
-static int locale_update_system_manager(sd_bus *bus, char **l_set, char **l_unset) {
+static int locale_update_system_manager(sd_bus *bus) {
```

Comment:
please rename to `reload_system_manager()` or so, since we are not longer updating the locale specifically, but reload the whole shebang